Peaky.Tests/TestClasses/AppleTests.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace Peaky.Tests.TestClasses
+{
+    public class AppleTests : IHaveTags
```

Comment:
Little classes like this that exist only to satisfy a few specific tests are typically placed closer to the tests that use them, since they have no meaning without that context.